UI/window-projector.cpp

```diff
@@ -32,7 +47,7 @@ OBSProjector::OBSProjector(QWidget *widget, obs_source_t *source_, int monitor,
 
 	// Mark the window as a projector so SetDisplayAffinity
 	// can skip it
-	windowHandle()->setProperty("isOBSProjectorWindow", true);
+	display.windowHandle()->setProperty("isOBSProjectorWindow", true);
```

Comment:
Not sure this line needs to change. I suspect it should probably be left alone.

Reply:
I agree. The property should be set on the window, otherwise the check which uses this property will fail.